build/iOS.props

```diff
@@ -0,0 +1,18 @@
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <PackageReference Include="System.Collections" Version="4.3.0" />
```

Comment:
Are you sure that we need those libraries from *NuGet*? They should be available as a part of BCL.